README.md

```diff
@@ -7,7 +7,7 @@
   &nbsp;
   <a href="https://opensource.org/licenses/Apache-2.0"><img src="https://img.shields.io/badge/License-Apache%202.0-green.svg"></a>
   &nbsp;
-  <a href="https://docs.oracle.org/en/java/11/"><img src="https://img.shields.io/badge/Java-%3D11-green.svg"></a>
```

Comment:
ArcadeDB is still compatible with Java11, but now runs on Java17 too.

Reply:
I considered what is more important earliest or latest, and figured the most recent (LTS). However I now put two buttons in the README, one for Java 11 and one for Java 17.